lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +599,10 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color) do
+    color |> IO.ANSI.format_fragment(true)
```

Comment:
I don't see any reason to not keep the check though. Are there times when IO.ANSI.enabled? returns false and we still want the output ansicodes?